apps/src/blockly/blockSvgUnused.js

```diff
@@ -0,0 +1,213 @@
+  });
+
+  var clip = Blockly.utils.dom.createSvgElement(
+    'clipPath',
+    {
+      id: `frameClip${this.block_.id}`
+    },
+    this.frameGroup_
+  );
+  this.frameClipRect_ = Blockly.utils.dom.createSvgElement(
+    'rect',
+    {
+      x: -FRAME_MARGIN_SIDE,
+      y: -(FRAME_MARGIN_TOP + FRAME_HEADER_HEIGHT),
+      height: FRAME_HEADER_HEIGHT
+    },
+    clip
+  );
+
+  this.frameBase_ = Blockly.utils.dom.createSvgElement(
+    'rect',
+    {
+      x: -FRAME_MARGIN_SIDE,
+      y: -(FRAME_MARGIN_TOP + FRAME_HEADER_HEIGHT),
+      fill: '#e7e8ea',
+      stroke: '#c6cacd',
+      rx: 15,
+      ry: 15
+    },
+    this.frameGroup_
+  );
+
+  this.frameHeader_ = Blockly.utils.dom.createSvgElement(
+    'rect',
+    {
+      x: -FRAME_MARGIN_SIDE,
+      y: -(FRAME_MARGIN_TOP + FRAME_HEADER_HEIGHT),
+      fill: '#c6cacd',
+      rx: 15,
+      ry: 15,
+      'clip-path': `url(#frameClip${this.block_.id.replace(/\)/g, '\\)')})`
+    },
+    this.frameGroup_
+  );
+
+  var frameTextVerticalPosition = -(FRAME_MARGIN_TOP + FRAME_HEADER_HEIGHT / 2);
+  if (Blockly.ieVersion()) {
+    // in non-IE browsers, we use dominant-baseline to vertically center
+    // our text. That is unfortunately not supported in IE, so we
+    // manually offset by 4 pixels to compensate.
+    frameTextVerticalPosition += 4;
+  }
+  this.frameText_ = Blockly.utils.dom.createSvgElement(
+    'text',
+    {
+      class: 'blocklyText',
+      style: 'font-size: 12pt;fill: black',
+      y: frameTextVerticalPosition,
+      'dominant-baseline': 'central'
+    },
+    this.frameGroup_
+  );
+  this.frameText_.appendChild(document.createTextNode(msg.unusedCode()));
+
+  this.frameHelp_ = Blockly.utils.dom.createSvgElement(
+    'g',
+    {
+      class: 'blocklyHelp'
+    },
+    this.frameGroup_
+  );
+  Blockly.utils.dom.createSvgElement(
+    'circle',
+    {
+      fill: '#7665a0',
+      r: FRAME_HEADER_HEIGHT * 0.75 * 0.5
+    },
+    this.frameHelp_
+  );
+  Blockly.utils.dom
+    .createSvgElement(
+      'text',
+      {
+        class: 'blocklyText',
+        y: Blockly.ieVersion() ? 4 : 0, // again, offset text manually in IE
+        'dominant-baseline': 'central',
+        'text-anchor': 'middle'
+      },
+      this.frameHelp_
+    )
+    .appendChild(document.createTextNode('?'));
+};
+
+/**
+ * @override
+ */
+BlockSvgUnused.prototype.getPadding = function() {
+  return {
+    top: FRAME_MARGIN_TOP + FRAME_HEADER_HEIGHT,
+    right: FRAME_MARGIN_SIDE,
+    bottom: FRAME_MARGIN_BOTTOM,
+    left: FRAME_MARGIN_SIDE
+  };
+};
+
+BlockSvgUnused.prototype.bindClickEvent = function() {
+  if (this.isbound_) {
+    return;
+  }
+  this.isbound_ = true;
+
+  // We bind to mousedown rather than click so we can interrupt the drag
+  // that would otherwise be initiated.
+  Blockly.bindEvent_(this.frameHelp_, 'mousedown', this, function(e) {
+    if (Blockly.isRightButton(e)) {
+      // Right-click.
+      return;
+    }
+
+    this.helpClickFunc_(e);
```

Comment:
`Blockly.fireUiEvent` doesn't exist anymore, which I think is going to be kind of a bummer for a lot of our customizations. In this case, it was pretty straightforward to modify to be able to call the callback directly. Yet to be seen whether that works for other uses.